helm/dagster/charts/dagster-user-deployments/values.yaml

```diff
@@ -155,6 +155,15 @@ serviceAccount:
 # Whether to bind a role to the service account that allows it to create Kubernetes jobs
 rbacEnabled: true
 
+extraInitContainers: []
+
+extraContainers: []
```

Comment:
maybe "sidecarContainers"?

Reply:
I will suggest called  them just initContainers and sidecarContainers.

Reply:
my names matched airflow's that are similar in purpose, but i'm open to change them to the suggested names.
https://airflow.apache.org/docs/helm-chart/stable/using-additional-containers.html